app/src/main/java/org/astraea/app/common/Utils.java

```diff
@@ -166,6 +166,40 @@ public static int requirePositive(int value) {
     return value;
   }
 
+  public static boolean isPositive(int value) {
+    return value > 0;
+  }
+
+  public static Object reflectionAttribute(Object object, String attribute) {
+    try {
+      var field = object.getClass().getDeclaredField(attribute);
+      field.setAccessible(true);
+      return field.get(object);
+    } catch (NoSuchFieldException | IllegalAccessException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  /**
+   * reflection inner class attribute
+   *
+   * @param object out class
+   * @param innerObjectName inner objectName
+   * @param attribute attribute name
+   * @return attribute
+   */
+  public static Object reflectionAttribute(
```

Comment:
Why we need `innerObjectName` here? We can get class info from `object directly.